src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py

```diff
@@ -1655,7 +1655,39 @@ def is_year_end():
 
     @property
     def is_leap_year():
-        pass
+        """
+        Boolean indicator if the date belongs to a leap year.
+
+        A leap year is a year, which has 366 days (instead of 365) including 29th of February as an intercalary day. Leap years are years which are multiples of four with the exception of years divisible by 100 but not by 400.
+
+        Returns
+        -------
+        Series or ndarray
+            Booleans indicating if dates belong to a leap year.
+
+        Examples
+        --------
+        This method is available on Series with datetime values under the .dt accessor, and directly on DatetimeIndex.
+
+        >>> idx = pd.date_range("2012-01-01", "2015-01-01", freq="YE")
+        >>> idx  # doctest: +SKIP
```

Comment:
why skip?

Reply:
`date_range` returns a `Series` and not a `DatetimeIndex` like native pandas. So the doctest would fail if not skipped. The other option is to change the result in the doctest to be `Series` instead of `DatetimeIndex`. But this is already covered in the following test case. I prefer to skip the `DatetimeIndex` case here - such that we can later come back and unskip it - once `date_range` can return `DatetimeIndex`. By the way, I think we should plan to  do this once we have a lazy implementation for `DatatimeIndex`.

Reply:
Please add a todo comment with this jira number https://snowflakecomputing.atlassian.net/browse/SNOW-1486910. We will fix it soon. Thanks!

Reply:
Done.